### ISSUE TITLE
Linode API v3 is no longer active. Please use Linode API v4

### ISSUE THREAD

**Issue by mstenta** [certbot/certbot#9753]
## My operating system is (include version):

Debian GNU/Linux 10 (buster)

## I installed Certbot with (snap, OS package manager, pip, certbot-auto, etc):

Docker

## I ran this command and it produced this output:

`docker run --rm --name certbot -v '[OMITTED]:/var/letsencrypt/linode.ini' -v '[OMITTED]:/etc/letsencrypt' -v '[OMITTED]:/var/www/letsencrypt' certbot/dns-linode renew --dns-linode-propagation-seconds 300`

Output:

`Failed to renew certificate [OMITTED] with error: Unexpected error determining zone identifier for [OMITTED]: Linode api error: [{'ERRORMESSAGE': 'Linode API v3 is no longer active. Please use Linode API v4: https://www.linode.com/docs/api', 'ERRORCODE': 16}]`

## Certbot's behavior differed from what I expected because:

Linode deprecated their v3 API in December (I think): https://www.linode.com/docs/products/tools/api/guides/upgrade-to-v4/

And it appears that they officially turned it off this past Saturday, August 19th. That's when my renewals started failing with the error above.

**Comment by mstenta** [comment]
I'm looking at the code in this repo and it *seems* like it handles API v4 already... :thinking:

https://github.com/certbot/certbot/blob/3e84b94308a9919ef9affe5dac8d5a65554cda67/certbot-dns-linode/certbot_dns_linode/_internal/dns_linode.py

So I'm not sure if there's something I need to reconfigure to make it work? This may just be a support request - sorry if this has been covered somewhere already. I searched the issue queue and https://community.letsencrypt.org/ but didn't see anything.

**Comment by osirisinferi** [comment]
Did you set `dns_linode_version` to "4" in your credentials file?

**Comment by mstenta** [comment]
Ah ha! I think I figured out the problem. I generated a new API key at https://cloud.linode.com/profile/tokens and replaced my old one, and it seems to be working now. :-)

> Did you set `dns_linode_version` to "4" in your credentials file?

No I didn't - thank you for the suggestion @osirisinferi. I guess that's optional, but I might add it anyway now just to be explicit.

I hope this helps anyone who runs into the same issue. :-)
